Command/PopulateCommand.php

```diff
@@ -91,7 +91,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $type = $input->getOption('type');
         $reset = !$input->getOption('no-reset');
         $options = array(
-            'batch_size' => $input->getOption('batch-size'),
+            'batch_size' => ( !empty($input->getOption('batch-size') ) ) ? $input->getOption('batch-size') : 1,
```

Comment:
this is not compatible with PHP 5.3 and 5.4 where ``empty`` can only be used on variables